runtime/lua/vim/lsp/client.lua

```diff
@@ -715,7 +715,7 @@ end
 --- @param params table LSP request params.
 --- @param timeout_ms integer? Maximum time in milliseconds to wait for
 ---                                a result. Defaults to 1000
---- @param bufnr integer Buffer handle (0 for current).
+--- @param bufnr? integer Buffer handle (0 for current).
```

Comment:
```suggestion
--- @param bufnr? integer (default: 0) Buffer handle, or 0 for current.
```

Reply:
The original purpose here is to be the same as the `Client:request` above, so I have pushed a change that simultaneously improves both places now.